src/eos-kolibri-manage.in

```diff
@@ -17,12 +17,29 @@ if [ ! -f "${KOLIBRI_HOME}/db.sqlite3" ]; then
   exit 1
 fi
 
+case "$1" in
+*"/"*)
+  FLATPAK_COMMAND="$1"
+  SUBCOMMAND=""
+  shift
+  ;;
+"listcontent")
+  FLATPAK_COMMAND="/app/bin/kolibri-listcontent.py"
+  SUBCOMMAND=""
+  shift
+  ;;
+*)
+  FLATPAK_COMMAND="/app/bin/kolibri"
+  SUBCOMMAND="manage"
+  ;;
+esac
+
 runuser -u @KOLIBRI_USER@ -- flatpak run \
   --no-desktop \
   --env=KOLIBRI_HOME="${KOLIBRI_HOME}" \
   --filesystem="${KOLIBRI_HOME}" \
-  --command=/app/bin/kolibri \
+  --command="${FLATPAK_COMMAND}" \
   @KOLIBRI_FLATPAK_ID@ \
-  manage \
+  "${SUBCOMMAND}" \
```

Comment:
If I run:

```
eos-kolibri-manage /app/bin/foo bar
```

This will run `/app/bin/foo` with two parameters:
- `""` (the empty string)
- `"bar"`

I don't think that's what you want.

Reply:
Thanks! Fixed now :)